src/Nancy.ViewEngines.Razor/CSharp/CSharpRazorViewRenderer.cs

```diff
@@ -55,10 +58,18 @@ public CSharpRazorViewRenderer()
             this.ModelCodeGenerator = typeof(CSharpModelCodeGenerator);
 
             this.Provider = new CSharpCodeProvider();
+        }
+
+        /// <summary>
+        /// Get the host
+        /// </summary>
+        public NancyRazorEngineHost GetHost(string viewLocation)
+        {
+            var host = this.engineHostFactory.Create(new CSharpRazorCodeLanguage(), viewLocation);
 
-            this.Host = new NancyRazorEngineHost(new CSharpRazorCodeLanguage());
+            host.NamespaceImports.Add("Microsoft.CSharp.RuntimeBinder");

```

Comment:
Will this work on Mono? I had a look at http://go-mono.com/status/status.aspx?reference=4.0&profile=4.0&assembly=Microsoft.CSharp and that page tells me no.